app/helpers/display_format_helper.rb

```diff
@@ -0,0 +1,23 @@
+module DisplayFormatHelper
+  def date_format(date)
+    DateTime.parse(date).strftime("%B %Y") unless date.nil?
+  end
+
+  def head_info_list_maker(info)
+    [info['format'].humanize, date_format(info['public_timestamp'])]
+  end
+
+  def link_format(link)
+    if link.start_with?("https://", "http://")
+      link
+    elsif link.start_with?("www.")
+      "https://#{link}"
+    else
+      "https://gov.uk" + link
+    end
+  end
+
+  def name(link)
+    link_format(link).gsub(%r{https?://[^/]+/}, '').tr('-', ' ').tr('/', ' / ')
```

Comment:
Is there a reason to derive the name from the link instead of using the page title returned from rummager? I imagine this might look a bit weird sometimes since information (like punctuation) is lost when turning a title into a slug.

Btw if you just need to get the relative path, you could use the URI module
https://ruby-doc.org/stdlib-2.4.1/libdoc/uri/rdoc/URI.html